appinventor/appengine/src/com/google/appinventor/client/TopToolbar.java

```diff
@@ -144,9 +151,9 @@
   public TopToolbar() {
     /*
      * Layout is as follows:
-     * +--------------------------------------------------+
-     * | Project ▾ | Connect ▾ | Build ▾| Help ▾| Admin ▾ |
-     * +--------------------------------------------------+
+     * +---------------------------------------------------------------+
+     * | Project ▾ | Connect ▾ | Build ▾| Help ▾| Admin ▾ | Dyslexic ▾ |
```

Comment:
I wonder if maybe it is better to have this called "Settings" as I expect that we will want to have more in the future, and I don't know that we want to consume more primary menu space for an option that only affects a small portion of app inventors.